datafusion/physical-expr/src/window/window_frame_state.rs

```diff
@@ -285,8 +301,16 @@ impl WindowFrameStateRange {
         } else {
             current_row_values
         };
-        // `BISECT_SIDE` true means bisect_left, false means bisect_right
-        bisect::<BISECT_SIDE>(range_columns, &end_range, sort_options)
+        let search_start = if SIDE {
+            last_range.start
+        } else {
+            last_range.end
+        };
+        let compare_fn = |current: &[ScalarValue], target: &[ScalarValue]| {
```

Comment:
See the above comment about https://docs.rs/arrow/31.0.0/arrow/compute/struct.LexicographicalComparator.html possibly being another way to improve performance